CHANGELOG.md

```diff
@@ -4,10 +4,60 @@ All notable changes to this project will be documented in this file.
-## [Unreleased]
+## [0.9.0] - 2016-08-22
 
 ### Added
-- Sniff to flag dynamic translatable strings and textdomains.
+- `WordPress.WP.I18n` sniff to the `WordPress-Core` ruleset to flag dynamic translatable strings and textdomains.
+- `WordPress.PHP.DisallowAlternativePHPTags` sniff to the `WordPress-Core` ruleset to flag - and fix - ASP and `<script>` PHP open tags.
+- `WordPress.Classes.ClassOpeningStatement` sniff to the `WordPress-Core` ruleset to flag - and fix - class opening brace placement.
+- `WordPress.NamingConventions.ValidHookName` sniff to the `WordPress-Core` ruleset to flag filter and action hooks which don't comply with the guideline of lowercase letters and underscores. For maintaining backward-compatibility of hook names an `additionalWordDelimiters` property can be added via a custom ruleset.
+- `WordPress.Functions.DontExtract` sniff to the `WordPress-Core` ruleset to flag usage of the `extract()` function.
+- `WordPress.PHP.POSIXFunctions` sniff to the `WordPress-Core` ruleset to flag usage of regex functions from the POSIX PHP extension which was deprecated since PHP 5.3 and removed in PHP 7.
+- `WordPress.DB.RestrictedFunctions` and `WordPress.DB.RestrictedClasses` sniffs to the `WordPress-Core` ruleset to flag usage of direct database calls using PHP functions and classes rather than the WP functions for the same.
+- Abstract `AbstractClassRestrictions` parent class to allow for easier sniffing for usage of specific classes.
+- `Squiz.Strings.ConcatenationSpacing`, `PSR2.ControlStructures.ElseIfDeclaration`, `PSR2.Files.ClosingTag`, `Generic.NamingConventions.UpperCaseConstantName` to the `WordPress-Core` ruleset.
+- Ability to add arbitrary variables to the whitelist via a custom ruleset property for the `WordPress.NamingConventions.ValidVariableName` sniff.
+- Ability to use a whitelist comment for tax queries for the `WordPress.VIP.SlowDBQuery` sniff.
+- Instructions on how to use WPCS with Atom and SublimeLinter to the Readme.
+- Reference to the [wiki](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/wiki) to the Readme.
+- Recommendation to also use the [PHPCompatibility](https://github.com/wimg/PHPCompatibility) ruleset to the Readme.
+
+### Changed
+- The minimum required PHP_CodeSniffer version to 2.6.0.
+- Moved the `WordPress.WP.PreparedSQL` sniff from `WordPress-Extra` to `WordPress-Core`.
+- `WordPress.PHP.StrictInArray` will now also flag non-strict usage of `array_keys()` and `array_search()`.
+- Added `_deprecated_constructor()` and `_deprecated_hook()` to the list of printing functions.
+- Added numerous additional functions to sniff for to the `WordPress.VIP.RestrictedFunctions` sniff as per the VIP guidelines.
+- Upped the `posts_per_page` limit from 50 to 100 `WordPress.VIP.PostsPerPage` sniff as per the VIP guidelines.
+- Added `__debugInfo` to the magic method whitelist for functions starting with double underscore in the `WordPress.NamingConventions.ValidFunctionName` sniff.
```

Comment:
Considering the double underscore check was added in this release (as per line 34), this particular change seems redundant?